### PR TITLE
In CreateInteractionChoiceSetRequest change logic of processing

### PR DESCRIPTION
**Pull request by okozlovlux**
result of application.FindChoiceSet
  - changed logic of processing result of app->FindChoiceSet

Related to APPLINK-26517

@LevchenkoS @AByzhynar @Kozoriz @vlantonov @anosach-luxoft  please review

### CODE REVIEW

**Comment by LevchenkoS** [comment]
@okozlovlux, reviewed.

**Comment by vlantonov** [comment]
@okozlovlux Reviewed

**Comment by AByzhynar** [comment]
@okozlovlux Please reorder words in Commit header into: 
`Change processing logic of app.FindChoiceSet result in CreateInteractionChoiceSetRequest`

**Comment by okozlovlux** [comment]
@AByzhynar changed in 28b36de

**Comment by AByzhynar** [comment]
Reviewed

**Comment by Kozoriz** [comment]
Reviewed

**Comment by anosach-luxoft** [comment]
Reviewed

File: src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc [hunk 1]
```diff
@@ -99,7 +99,9 @@ void CreateInteractionChoiceSetRequest::Run() {
           .asInt();
 
   if (app->FindChoiceSet(choice_set_id_)) {
-    SDL_ERROR("Choice set with id " << choice_set_id_ << " is not found.");
+    SDL_ERROR("Choice set with id " << choice_set_id_
```
Comment (pestOO) [inline src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc]: @okozlovlux, 
Seem to be a Warning not an Error - please, see [Operational View -> Diagnostics](https://adc.luxoft.com/confluence/pages/viewpage.action?pageId=280330801#SoftwareArchitectureDocument%28SAD%29-OperationalView)

Comment (okozlovlux) [inline src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc]: @pestOO fixed 496720f

**Comment by wolfylambova22** [comment]
Reviewed.

**Pull request merged**